helpers/response/create_response.py

```diff
@@ -1,52 +1,51 @@
 from datetime import datetime
-from graphql import GraphQLError
+# from graphql import GraphQLError
 from api.response.models import Response
 from api.room_resource.models import Resource
-from utilities.validations import (
-    validate_rating_field,
-    validate_missing_items_field)
 
 
-def create_response(question_type, **kwargs):
+def create_response(question_type, errors, responses, **kwargs):
     if question_type.lower() == 'rate':
-        if 'rate' not in kwargs:
-            raise GraphQLError("Provide a rating response")
-        else:
-            validate_rating_field(**kwargs)
+        if 'rate' in kwargs and kwargs['rate']:
+            rating = [1, 2, 3, 4, 5]
```

Comment:
Why do you need to hard code the ratings here, can you not get the valid ratings fro the rating model?

Reply:
The ratings were hardcoded in the validations file before. I just moved it here.